### ISSUE TITLE
Caught exception: Error: Cannot find module 'lru-cache'

### ISSUE THREAD

**Issue by leighli** [ionic-team/ionic-cli#1382]
Hi 
I tried: ionic start projectname sidemenu

and received the Error:

Error: Cannot find module 'lru-cache'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at localpathdeets\npm\node_modules\ionic\node_modules\ionic-app-lib\node_modules\archiver\node_modules\file-utils\node_modules\glob\node_modules\minimatch\minimatch.js:35:11
    at Object.<anonymous> (localpathdeets\npm\node_modules\ionic\node_modules\ionic-app-lib\node_modules\archiver\node_modules\file-utils\node_modules\glob\node_modules\minimatch\minimatch.js:1057:3)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)

help please????

**Comment by jthoms1** [comment]
Could you provide me with the output from `ionic info` from within your project directory? Thanks, this should help in troubleshooting.

**Comment by SliceofCheese** [comment]
The issue with this error, is that you cannot actually use the "ionic info" command when this sort of error is occurring. The error seems to take priority over the ability to get info. 

 I had the same issue with "Cannot find module 'rx-lite-aggregates'" after subsequently fixing the "Cannot find lru-cache." 

What I had to do was actually uninstall ionic locally, and then reinstall each of these individually. Somehow something had gotten screwy between my github repo and the local files. So it turned out that I had *not* fully committed all of my files, and then later grabbed the repo on another computer, and some of the files were missing which is what caused this error to manifest. 

Try this: 

1) npm uninstall ionic 
2) npm install lru-cache

From there, you'll probably see that 'lru-cache' isn't the only file that you're missing. Just continue to install each missing module. It's also worth mentioning that I did update my typescript as well-- though I'm fairly certain that this isn't a necessarily step to fix the problem; it's just worth mentioning.
